daprdocs/content/en/developing-applications/building-blocks/pubsub/howto-publish-subscribe.md

```diff
@@ -342,6 +342,13 @@ app.post('/dsstatus', (req, res) => {
 
 {{< /tabs >}}
 
+
+## Content Types
```

Comment:
Can you put this heading/section at the start of the document after the Introduction and before the Steps. That way people read this before starting the steps, which show examples of the content-type in use.

Reply:
Done. added more content.